qiskit/aqua/operators/gradients/qfi_base.py

```diff
@@ -0,0 +1,68 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The module for Quantum the Fisher Information."""
+
+from typing import Union
+
+from qiskit.aqua.operators.gradients import DerivativeBase, CircuitQFI
+
+
+class QFIBase(DerivativeBase):  # pylint: disable=abstract-method
```

Comment:
Why is this `disable` needed?

Reply:
@t-imamichi

Reply:
I don't know the reason. Pylint passes without this pylint option in my environment. It is added by @manoelmarques. Could you tell us the reason, @manoelmarques?
https://github.com/Zoufalc/qiskit-aqua/commit/7f573596c7a281874b37416174c57e81c1ecf3f5

Reply:
We test againt two versions of pylint. The latest one and the one used by qiskit-terra. You probably have the latest version where the disable is not needed.

Reply:
Wouldn't it be easier to have the same version for all? Didn't fix pin the version some time ago to have the same as terra? 😄